### ISSUE TITLE
Getting error when building: failed to extract shortcode: template for shortcode "content-stip-left" not found

### ISSUE THREAD

**Issue by meggenberger** [zerostaticthemes/hugo-hero-theme#62]
Hi.

I followed the instructions for the installation on my Mac (M1) with Hugo extended installed (Hugo v0.115.2+extended darwin/arm64 BuildDate=unknown)

When I enter
hugo
to create the site I get the error

```
Start building sites …
hugo v0.115.2+extended darwin/arm64 BuildDate=unknown

Total in 22 ms
Error: error building site: assemble: "/Users/me/Code/Skarp/hero-theme/content/pages/about/index.md:11:1": failed to extract shortcode: template for shortcode "content-strip-left" not found
```

Any idea what I'm doing wrong?

**Comment by fschimpf** [comment]
Hi! 
I had the same problem. I deleted the file hugo.toml from the site root directory. After that everything works.

**Comment by JugglerX** [comment]
@meggenberger I can't reproduce the issue. I can see the shortcode does exist `layouts/shortcodes/content-strip-left.html` 

@fschimpf  I'm not sure what file you are referring too, there is not `hugo.toml` file.

I need more information to continue debugging.

**Comment by lgwapnitsky** [comment]
if you copy the files over from a new hugo site, the hugo.toml file exists. I can verify that, once that file is deleted, a hugo build works

**Comment by JugglerX** [comment]
Can someone provide exact steps? I dont see a hugo.toml file. Maybe you can link to it in the repo.

**Comment by richtong** [comment]
Yes, it you do not need to do the init and the cp -a does not remove it, I can submit a PR to remove the README.md line
